core/src/main/java/org/pentaho/di/core/row/ValueDataUtil.java

```diff
@@ -949,7 +973,7 @@ public static Object percent2( ValueMetaInterface metaA, Object dataA, ValueMeta
       case ValueMetaInterface.TYPE_BIGNUMBER:
         return metaA.getBigNumber( dataA ).subtract(
           divideBigDecimals( multiplyBigDecimals(
-            metaB.getBigNumber( dataB ), metaA.getBigNumber( dataA ), null ), new BigDecimal( 100 ), null ) );
+            metaB.getBigNumber( dataB ), metaA.getBigNumber( dataA ), null ), new BigDecimal( 100 ) ) );
```

Comment:
Need to put this back - cannot change the API

Reply:
Done